src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/entity/induction/PreviousWorkExperiencesEntity.kt

```diff
@@ -56,7 +56,7 @@ class PreviousWorkExperiencesEntity(
   var hasWorkedBefore: HasWorkedBefore? = null,
 
   @Column
-  val hasWorkedBeforeNotRelevantReason: String? = null,
+  var hasWorkedBeforeNotRelevantReason: String? = null,
```

Comment:
Subtle fix here - as well as not being mapped (🙄 ), the field in the entity needs to be a `var`, not a `val` so that the mapper can set it